libs/state/selections/src/lib/operators/select.ts

```diff
@@ -150,11 +202,25 @@ export function select<T,
  * @internal
  */
 export function select<T>(
-  ...opOrMapFn: OperatorFunction<T, any>[] | string[]
+  ...opOrMapFn:
+    | OperatorFunction<T, any>[]
+    | string[]
+    | [string, (val: any) => any]
+    | [string[], (slice: any) => any]
```

Comment:
What about using `unknown` instead of `any` here?

Reply:
i would prefer it as well if we are unable to provide a meaningful type

Reply:
Done :+1: